inc/taihostif.h

```diff
@@ -40,6 +40,17 @@ typedef enum _tai_host_interface_tx_align_status_t
     TAI_HOST_INTERFACE_TX_ALIGN_STATUS_DESKEW_LOCK
 } tai_host_interface_tx_align_status_t;
 
+/** @brief The host interface signal rate */
+typedef enum _tai_host_interface_signal_rate_t
+{
+    TAI_HOST_INTERFACE_SIGNAL_RATE_UNKNOWN,
+    TAI_HOST_INTERFACE_SIGNAL_RATE_100_GbE,
```

Comment:
nit: GbE to GBE to follow the convention

Reply:
I got it. I have done.